### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by skykataoka** [johngodley/redirection#706]
```
Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 5.6.30
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://perterminced.wp.xdomain.jp/wp-json/

Error: Failed to fetch

Error: Failed to fetch
```

**Comment by mawaw1** [comment]
same problem :(

**Comment by johngodley** [comment]
This should now be fixed in 3.1.1
